CONTRIBUTING.md

```diff
@@ -1,24 +1,58 @@
-# Contributing
+## Contributing
```

Comment:
any reason this shouldn't be a Heading 1, and the following to be Heading 2 (vs. what you have now as Heading 2 and 3 respectively)
```suggestion
# Contributing
```